src/test/functional/features/marriage-broken-down.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Marriage Broken Down
+
+  Background:
+    Given I am on divorce homepage
+    And I am on the admin portal sign in page
+    When I fill in the Username and Password fields with a valid login
+    And I click "Sign in"
+    And I see the divorce homepage
+    And I go to '/screening-questions/has-relationship-broken'
```

Comment:
This URL has been updated to match the requirements in #194
```suggestion
    And I go to '/irretrievable-breakdown'
```

Reply:
will be fixed once i merge master